lib/ReactViews/StandardUserInterface/standard-user-interface.scss

```diff
@@ -19,7 +19,7 @@
   }
 
   .top-element {
-    z-index: 99999 !important;
+    z-index: $notification-window-z-index - 10 !important;
```

Comment:
Now I see.. will need some consolidation as they are worlds apart
$front-component-z-index is 99
$notification-window-z-index is 99999
Also related: https://github.com/TerriaJS/terriajs/issues/3958